### DISCUSSION TITLE
After loaded a backup save and docked into next outpost, game loaded into the backup save that previous loaded

### DISCUSSION THREAD

**Discussion by 1170991249** [FakeFishGames/Barotrauma#15056] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

First time, our last save crashed so we loaded into a backup save A, the sever crashed at the mid of the round and the server loaded back to backup save A that is still docked in the outpost, character items also loaded into the backup save A
Second time, we succesfully docked into the next outpost, the server still loded into backup save A, but with character items keeped

### Reproduction steps

1.Load into a backup save
2.Dock into the next station
3.Server loaded into previous backup save

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/issues/6008 (PR in our private repo). Fix coming up in the next unstable build + future update!
